.github/workflows/deploy.yml

```diff
@@ -8,27 +8,24 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-latest
-  
+ 
     steps: 
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
     - name: Setup CD Environment
-      uses: yetanalytics/actions/setup-env@v0
-    
+      uses: yetanalytics/actions/setup-env@v0.0.1
```

Comment:
Is there a reason _not_ to use the latest version if you're gonna be changing it anyhow?

Reply:
That..was not on purpose lol.

Reply:
https://github.com/yetanalytics/colossal-squuid/pull/17/commits/176c6711c6f5f8b6327448f51b494da8dd1b1845